sphinx/writers/latex.py

```diff
@@ -399,6 +401,15 @@ def __init__(self, document, builder):
         if self.elements['polyglossia']:
             self.elements['babel'] = ''  # disable babel forcely
             self.elements['multilingual'] = self.elements['polyglossia']
+            self.elements['classoptions'] += ',' + self.babel.get_language()
```

Comment:
from short look at current polyglossia code, it does not do anything with document class options; furthermore it does not accept the language as package options (but older polyglossia) did. It is needed to use `\setmainlanguage` or `\setdefaultlanguage` which are synonyms.

If I understand correctly moving

     self.elements['classoptions'] += ',' + self.babel.get_language()

to before conditional will simply result in an added ``,`` in class options if neither babel nor polyglossia is enabled ? It is ok for LaTeX to  have ``,,`` in the document class options, so yes I think it is better to move this line before the if-block.